src/vl-tooltip.js

```diff
@@ -253,6 +251,14 @@
         default:
       }
     }
+    
+    // supplement bin from fieldDef, user should never have to provide bin
+    if (fieldDef.bin) {
+      supplementedFieldOption.bin = true;
+      supplementedFieldOption.formatType = 'string'; // we show bin range as string (e.g. "5-10")
```

Comment:
is formatType `string` really required?  

(This seems inconsistent with the rest of Vega universe)